tests/functional/image-uploads-non-auth.js

```diff
@@ -7,7 +7,9 @@ const intern = require("intern").default;
 const { assert } = intern.getPlugin("chai");
 const { registerSuite } = intern.getInterface("object");
 const FunctionalHelpers = require("./lib/helpers.js");
-const url = intern.config.functionalBaseUrl + "issues/new";
+const baseUrl = intern.config.functionalBaseUrl;
+const url = baseUrl + "issues/new";
+const targetOrigin = baseUrl.slice(0, baseUrl.length - 1);
```

Comment:
hum, why? to remove the last 0 or maybe an `/` ?

```
"http://localhost:5000".slice(0, "http://localhost:5000".length -1)

-> "http://localhost:500"
```

Reply:
`"functionalBaseUrl": "http://localhost:5000",` : https://github.com/webcompat/webcompat.com/blob/master/intern.json#L5

Reply:
yeah, so this is kind of weird, but when you access the value intern adds a terminating `/` -> `http://localhost:5000/`, and postMessage won't match on that. 

(maybe i should file a bug with intern...)